digdag-ui/model.js

```diff
@@ -297,6 +297,15 @@ export class Model {
     })
   }
 
+  retrySession (session: Session) {
+    const { lastAttempt } = session
+    return this.put('attempts', {
```

Comment:
in which case why not use the server generated one? (as the code does right now)